.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
Please don't add a `.gitignore`: these files should be in your global gitignore.